### ISSUE TITLE
Regexp starting with = doesn't get highlighted.

### ISSUE THREAD

**Issue by envygeeks** [atom/language-ruby#66]
When doing the regexp:

``` ruby
/=\Z/
```

it doesn't get highlighted.  It seems to only happen with `=`

**Comment by nokutu** [comment]
It gets highlighted if you write:
```
/\=\Z/
```
But I think both are the same valid regular expression.

**Comment by 50Wliu** [comment]
Ugh, this is annoying:
```cson
{
    'comment': 'Needs higher precidence than regular expressions.'
    'match': '(?<!\\()/='
    'name': 'keyword.operator.assignment.augmented.ruby'
}
```

No more easy fix :(

**Comment by envygeeks** [comment]
`/=` is not valid Ruby, from what I gander it's trying to test for `[]=(` as in:

```ruby
def []=(key, val)
  $stdout.puts key, val
end
```

**Comment by envygeeks** [comment]
Well... supposed to be trying to match it, I don't know why the `/` is there. Seems like a bug to me.

**Comment by 50Wliu** [comment]
Since the match is `keyword.operator.assignment.augmented` I believe it is actually trying to match `/=` (an "augmented" operator).

If it's not valid though...hopefully should be a simple deletion!

**Comment by envygeeks** [comment]
@50Wliu you're right, I forgot about that since like literally nobody ever uses it, actually needs to be a var so you'll have to do something like `x = 120; x /= 12`  It's for assigning the result of the division.  You can actually do it with all math operators:  `**=`, `*=`, `/=`, `+=`, `-=`

**Comment by envygeeks** [comment]
> actually needs to be a var 

I should probably rephrase, it needs to have a setter.
